massa-protocol-worker/src/ip.rs

```diff
@@ -0,0 +1,14 @@
+use std::net::IpAddr;
+
+// TODO: Use std one when stable
+pub(crate) fn to_canonical(ip: IpAddr) -> IpAddr {
```

Comment:
this is not included in the ip_rfc crate ?

Reply:
No but this is the exact same code as the std one we can use it and it's pretty light